index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nove Theme</title>
+    <!-- Favicon -->
+    <link rel="shortcut icon" href="./images/favicon.png" type="image/x-icon">
+    <!-- Styles Links -->
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/unit.css">
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="stylesheet" href="./styles/fonts.css">
+
+</head>
+<body>
+<!-- Header Section -->
+    <header>
+        <section class="row header__section">
+            <nav>
+                <div class="container">
+
+                    <!-- Nav Logo -->
+                    <div class="cell logo">
+                        <img src="./images/logo.svg" alt="Nova Logo">
+                    </div>
+
+                    <!-- Nav Menu -->
+                    <ul class="cell nav__list">
+                        <li class="nav__item"><a href="#">The Team</a></li>
+                        <li class="nav__item"><a href="#">Pricing</a></li>
+                        <li class="nav__item"><a href="#">Features</a></li>
+                    </ul>
+                </div>
+            </nav>
+
+            <!-- HEADER TEXT -->
+            <div class="container">
+                <div class="cell header_mainText">
```

Comment:
Skoro `mainText` to może użyć tutaj `<article>`?

PS. Raczej w BEM używamy `-` do rozdzielania wyrazów tj. `main-text`